.github/workflows/build-test-riscv64.yml

```
@@ -0,0 +1,59 @@
+name: Build and test riscv64 on ubuntu-latest
+
+on:
+  push:
+    branches:
+      - main
+      - dev
+    tags:
+        - '**'
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  build:
+    timeout-minutes: 60
+    name: QEMU riscv64 via Debian on ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 1
+
+    - name: Set up QEMU on x86_64
+      if: startsWith(matrix.os, 'ubuntu-latest')
+      id: qemu
+      uses: docker/setup-qemu-action@v2
+      with:
+        platforms: riscv64
+#        image: tonistiigi/binfmt:latest
+
+    - name: Build and Test
+      run: |
+        docker run --rm \
```

```suggestion
        docker run --rm --platform linux/riscv64 \
```

I think you get it implicitly because of the image, but usually prefer to specify `--platform linux/riscv64` as well